front/src/applications/stdcm/components/StdcmResults/StdcmDebugResults.tsx

```diff
@@ -73,7 +73,6 @@ const StdcmDebugResults = ({ simulationOutputs }: StdcmDebugResultsProps) => {
                 <ManchetteWithSpaceTimeChartWrapper
                   operationalPoints={pathProperties.manchetteOperationalPoints}
                   projectPathTrainResult={projectedData.spaceTimeData}
-                  selectedTrainScheduleId={STDCM_TRAIN_TIMETABLE_ID}
```

Comment:
You should keep it as selectedTrainId no ?
```suggestion
                  selectedTrainId={STDCM_TRAIN_TIMETABLE_ID}
```

Reply:
done